README.md

```diff
@@ -2,6 +2,22 @@
 
 `skynet-accounts` is a service implementing user accounts for [Skynet](https://siasky.net) portals. It uses MongoDB for data storage.
 
+## Setup steps
+
+### The `.env` file
+
+All local secrets are loaded from a `.env` file in the root directory of the project.
+
+Those are (example values):
+```.env
+SKYNET_DB_HOST="localhost"
+SKYNET_DB_PORT="27017"
+SKYNET_DB_USER="username"
+SKYNET_DB_PASS="password"
+SKYNET_PEPPER="some random string for pepper"
```

Comment:
sorry but what is "pepper".. if that's some joke on a salt we should definitely get rid of it

Reply:
Oh, you and your unrelenting hatred towards anything fun!

https://en.wikipedia.org/wiki/Pepper_(cryptography)